met-web/src/components/engagement/new/create/index.tsx

```diff
@@ -49,6 +51,7 @@ const EngagementCreationWizard = () => {
     const [open, setOpen] = React.useState(true);
     const navigate = useNavigate();
     const fetcher = useFetcher();
+    const { languages } = useRouteLoaderData('wizard-loader') as { languages: Language[] };
```

Comment:
Since we're on the route we're loading from, consider using `useLoaderData()` instead :)